src/java.desktop/macosx/native/libawt_lwawt/awt/ApplicationDelegate.m

```diff
@@ -283,8 +291,10 @@ - (void)_handleOpenURLEvent:(NSAppleEventDescriptor *)openURLEvent withReplyEven
     //fprintf(stderr,"jm_handleOpenURL\n");
     JNIEnv *env = [ThreadUtilities getJNIEnv];
     jstring jURL = JNFNSToJavaString(env, url);
```

Comment:
Don't we need to remove this JNFNSToJavaString dependancy?

Reply:
Not for these changes. There will be a subsequent patch to update all of the string conversions